docs/02-guides/override-a-component.mdx

```diff
@@ -2,14 +2,177 @@
 title: Override a component
 description:
   This guide explains how to override a component in Front-Commerce, to
-  customize the default theme
+  customize the chocolatine theme
```

Comment:
I think it would be better to keep it as `theme-chocolatine`. This will help with keywords `@front-commerce/theme-chocolatine` and also be more consistent in the rest of the documentation

Reply:
Although I think this should not reference any theme, because the override a component is not only for theme-chocolatine, it can be for any future themes, so I would keep it as default theme. The default theme here is referencing the default used by the integrators, either theme-chocolatine, or any other future theme.

Reply:
I agree with both statements, I'd go with something among those lines:

```suggestion
  customize the theme of your application
```

Reply:
fixed with @Kagaminara suggestion